hub.json

```diff
@@ -28,7 +28,9 @@
     "cerebriumai": [
         "airbyte_dbt_github",
         "airbyte_dbt_google_ads",
-        "airbyte_dbt_shopify"
+        "airbyte_dbt_shopify",
+        "airbyte_dbt_intercom",
+        "airbyte_dbt_stripe"
```

Comment:
I see the intercom package in the list of public repos on GitHub. But I'm not seeing the stripe one. Is it possible is it categorized as private rather than public?

Here's where I'm looking, and I only see 8 public packages:
https://github.com/orgs/CerebriumAI/repositories